SD DB Setup.sql

```diff
@@ -422,7 +422,7 @@ create table voting_rights
     voting_notes text,
     pending_leg text,
     date_modified timestamp not null default current_timestamp(),
-    user_modified varchar(20) not null);
+    user_modified varchar(20) default null);
     
```

Comment:
Wouldn't we want the user_modified column to be not null? If there is a trigger it shouldn't matter as it would be populated at insert time.